core/domain/user_services.py

```diff
@@ -388,3 +391,102 @@ def get_email_preferences(user_id):
             if email_preferences_model is None
             else email_preferences_model.site_updates)
     }
+
+
+class UserContributions(object):
+    """Value object representing a user's contributions."""
+    def __init__(
+            self, user_id, created_explorations=None, edited_explorations=None):
+        self.user_id = user_id
+        self.created_explorations = created_explorations
+        self.edited_explorations = edited_explorations
+
+
+    def validate(self):
+        if not isinstance(self.user_id, basestring):
+            raise utils.ValidationError(
+                'Expected user_id to be a string, received %s' % self.user_id)
+        if not self.user_id:
+            raise utils.ValidationError('No user id specified.')
+
+
+def get_user_contributions(user_id, strict=False):
+    """Gets domain object representing the contributions for the given user_id.
+
+    If the given user_id does not exist, returns None.
+    """
+    model = user_models.UserContributionsModel.get(user_id, strict=False)
+    if model is not None:
+        result = UserContributions(
+                    model.id, created_explorations=model.created_explorations,
+                    edited_explorations=model.edited_explorations)
+    else:
+        result = None
+    return result
+
+
+def create_user_contributions(user_id, created_explorations, edited_explorations):
+    """Creates a new UserContributionsModel and returns the domain object."""
+    user_contributions = get_user_contributions(user_id, strict=False)
+    if user_contributions is not None:
+        raise Exception('User %s already exists.' % user_id)
+    else:
+        user_contributions = UserContributions(
+            user_id=user_id, created_explorations=created_explorations, 
```

Comment:
Don't need arg names for positional args. I.e. (assuming the default args for the other fields are removed too) I would expect to see

    UserContributions(user_id, created_explorations, edited_explorations)